src/member.spec.ts

```diff
@@ -31,6 +31,7 @@ describe('member', () => {
   let connection: Connection;
   let serviceOptions: ServiceOptions;
   let memberModel: ReturnModelType<typeof MemberModelSchema>;
+  const mongoId = '6424ad273f75645b35f9ee79';
```

Comment:
I think it's confusing to define `mongoId` here, when it's actually only needed in the non-existing case. 

`Record Find errors should be propagated` creates an actual record, so we could/should just use that one.